### DISCUSSION TITLE
(Unofficial) PSA: Possible Scammers contact contributors of mkdocs-material through mail

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#6992] [General]
I'm posting this here, as I think it's important to share and to spread awareness.

I've received an e-mail from a random address where I was congratulated for my "contributions to squidfunk/mkdocs-material".
They also mentioned that I have the chance to receive "Starknet tokens" (which from what I gather is yet another crypto) by doing some stuff.
This is with 120% certainty a scam and will most likely end with you at the very least receiving further spam, or in worst have your crypto wallet drained.

Please do NOT fall for these mails. They are most certainly not legit in any way. The fact that I of all people received this and that it mentions this repository makes me believe, that they went through contributors to this repository (Or recent discussion authors) and contact everyone with a public e-mail.

Stay safe!

**Comment by alexvoss** [comment]
Yep, got one of those and it went the same way as the $200M from a late Nigerian prince that his niece contacted contacted me about.

**Reply by facelessuser** [reply]
You know you've made it when people are specifically targeting users or developers of your product.
